Karma/Migrations/KarmaContextModelSnapshot.cs

```diff
@@ -115,8 +114,7 @@ protected override void BuildModel(ModelBuilder modelBuilder)
                     b.Property<bool>("IsSeenByPostUser")
                         .HasColumnType("INTEGER");
 
-                    b.Property<string>("PostUserId")
-                        .IsRequired()
+                    b.Property<string>("PostUserIdId")
```

Comment:
Why is Id repeated 2 times?

Reply:
I forgot `add-migration` automatically adds suffix 'id' to a field with a foreign entity. Thanks for noticing, I will rename `PostUserId` to `PostUser`.